lib/proxy/helpers.rb

```diff
@@ -83,6 +83,9 @@ def remote_fqdn(forward_verify = true)
     ip = request.env['REMOTE_ADDR']
     log_halt 403, 'could not get remote address from environment' if ip.empty?
 
+    # IPv6 can contain %eno1 (NIC identifier) so strip it
+    ip = ip.sub(/%.*$/, '')
```

Comment:
I think we should return if the IP contains `%`. I don't think we'll find a reverse DNS for link local addresses and if we do, can we really trust it?

Reply:
Yeah makes sense, rebased.